<?xml version="1.0"?>
<!--
使用范例:
 $roslaunch mot_oc mot_oc.launch
-->
<launch>
  <!-- 节点名称 -->
  <arg name="node_name" default="dl_vis" />
  <!-- 订阅主题: 图像数据 -->
  <arg name="tp_in_image" default="/fl_cam/color/image_raw" />
  <!-- 订阅主题: 语义分割 -->
  <arg name='tp_in_seg' default="/segmentation" />
  <!-- 发布主题: 可视化检测结果 [sensor_msgs/Image] -->
  <arg name="tp_out_vis" default="/dl_vis" />
  <!-- <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=/fl_cam/color raw out:=/fl_cam/color/image_raw" /> -->
  <node pkg="dl_vis" type="dlvis_node.py" name="$(arg node_name)" output="screen">
      <param name='tp_in_image' value="$(arg tp_in_image)" />
      <param name='tp_in_seg' value="$(arg tp_in_seg)" />
      <param name="tp_out_vis" value="$(arg tp_out_vis)" />
  </node>
</launch>

